com.gluon.scenebuilder.yml

```diff
@@ -0,0 +1,56 @@
+app-id: com.gluon.scenebuilder
```

Comment:
I will reiterate: this app-id is INVALID.

Reply:
updated to com.gluonhq.scenebuilder

Reply:
updated to com.gluonhq.scenebuilder